rllib/algorithms/algorithm_config.py

```diff
@@ -3206,24 +3226,15 @@ def get_learner_group_config(self, module_spec: ModuleSpec) -> LearnerGroupConfi
         if not self._is_frozen:
             raise ValueError(
                 "Cannot call `get_learner_group_config()` on an unfrozen "
-                "AlgorithmConfig! Please call `freeze()` first."
+                "AlgorithmConfig! Please call `AlgorithmConfig.freeze()` first."
             )
 
         config = (
             LearnerGroupConfig()
             .module(module_spec)
             .learner(
                 learner_class=self.learner_class,
-                # TODO (Kourosh): optimizer config can now be more complicated.
-                # TODO (Sven): Shouldn't optimizer config be part of learner HPs?
-                #  E.g. if we have a lr schedule, this will have to be managed by
-                #  the learner, NOT the optimizer directly.
-                optimizer_config={
-                    "lr": self.lr,
-                    "grad_clip": self.grad_clip,
-                    "grad_clip_by": self.grad_clip_by,
-                },
-                learner_hyperparameters=self.get_learner_hyperparameters(),
+                learner_hyperparameters=self.get_learner_hyperparameters(module_spec),
```

Comment:
no more optimizer_config! All config information is now passed via `LearnerHyperparameters` to the Learner.